src/engine/renderer/tr_init.cpp

```diff
@@ -1107,7 +1107,16 @@ ScreenshotCmd screenshotPNGRegistration("screenshotPNG", ssFormat_t::SSF_PNG, "p
 		r_exportTextures = ri.Cvar_Get( "r_exportTextures", "0", 0 );
 		r_heatHaze = ri.Cvar_Get( "r_heatHaze", "1", CVAR_LATCH | CVAR_ARCHIVE );
 		r_noMarksOnTrisurfs = ri.Cvar_Get( "r_noMarksOnTrisurfs", "1", CVAR_CHEAT );
-		r_lazyShaders = ri.Cvar_Get( "r_lazyShaders", "0", 0 );
+
+		/* Delay GLSL shader build until first map load.
```

Comment:
It's not clear that this comment is talking about the effect of the "1" value of the cvar.

Otherwise LGTM

Reply:
I updated the comment. I also switched target branch to `master` as it is compatible with master and it makes easier to test it with public games.